sdk/storage/storage-blob/rollup.base.config.js

```diff
@@ -53,7 +53,11 @@ export function nodeConfig(test = false) {
       }),
       nodeResolve({ preferBuiltins: true }),
       cjs()
-    ]
+    ],
+    onwarn(warning, warn) {
```

Comment:
I wonder if we should pull out these common configurations to some central place, it's a shame to have to duplicate it in every rollup config. Maybe this is just an item to make a common rollup config.

Reply:
I'm working on this as we speak. Idea is either to create a custom rollup plugin or just a function that builds the canonical rollup config from high level configurations like library name.

Reply:
Good to know :)